### PR TITLE
Add note about detailed changelog

### PR DESCRIPTION
**Pull request by adamdruppe**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ Digger will automatically bisect the history for you.
 We use Bugzilla to list fixed issues on a new release.  This list is then included in the changelog.
 For this list to be accurate then invalid or duplicated bugs must be closed with the appropriate resolution ('RESOLVED INVALID' and 'RESOLVED DUPLICATE', respectively - as opposed to e.g. 'RESOLVED FIXED').
 
+For larger changes, you may want to add a specialized, detailed entry as a file in the `changelog/` directory. See [changelog/README.md](changelog/README.md) for details.
+
```
Comment (Geod24) [inline CONTRIBUTING.md:39]:
```suggestion
Changes that don't come with a bugzilla entry, such as new features, or enhancement on existing features, should come with their own changelog entry. Changelog entries should be written for everyday users, not compiler contributors.
Additionally, for larger changes, you may want to add a specialized, detailed entry even if a bugzilla entry exists.
See [changelog/README.md](changelog/README.md) for details on how to add a changelog entry.
Note that after a version has entered the release window (there is a beta / the change is in stable), changes should be made directly [in the dlang.org repository](https://github.com/dlang/dlang.org/tree/master/changelog).
```

**Comment by Geod24** [comment]
Sorry for the full rephrase, tried to add more details / context, hope it's not too verbose.

**Comment by adamdruppe** [comment]
my "for" survived!

I just want the link to be there, I didn't even know that other thing existed.

**Comment by adamdruppe** [comment]
whoop by rewriting history i destroyed your authorship note

but meh team effort anyway

**Comment by Geod24** [comment]
> whoop by rewriting history i destroyed your authorship note

Yeah no worries :)

**Comment by ibuclaw** [comment]
Not that anybody reads it, but could be added to a new PR template? Or am I imagining those exist.

**Comment by PetarKirov** [comment]
@ibuclaw there are pull request description templatea:
https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

**Comment by Geod24** [comment]
For ref they can be added here to affect the whole org: https://github.com/dlang/.github

**Pull request merged**